src/main.py

```diff
@@ -636,6 +623,7 @@ def replica_return(self, db_state, zk_state):
         my_hostname = helpers.get_hostname()
         self.write_host_stat(my_hostname, db_state)
         holder = zk_state['lock_holder']
+        logging.debug('Replica was return. So we resume WAL replay to {}'.format(holder))
```

Comment:
`Replica is returning`?
And it looks like logging an intent, shouldn't it be marked with `ACTION` and placed right before that action (https://github.com/yandex/pgconsul/pull/59/files#diff-2e5ad92c43aa96cc3a9cef6c6aec998b216f1379c43b1f651013d25e55989312R632)?